go.mod

```diff
@@ -2,24 +2,33 @@ module github.com/openshift/ptp-operator
 
 go 1.18
 
+replace k8s.io/client-go => k8s.io/client-go v0.25.2
```

Comment:
Why are these specific versions required?

Reply:
fixed, was due to a pevious conflict